app/helpers/application_helper.rb

```diff
@@ -1,10 +1,10 @@
 module ApplicationHelper
-	def full_title  page_title=''
-		base_title="Ruby on Rails Tutorial Sample App"
-		if page_title.empty?
-			base_title
-		else
-			page_title+ "|" +base_title
-		end
-	end
+  def full_title  page_title=""
+    base_title = "Ruby on Rails Tutorial Sample App"
+    if page_title.empty?
```

Comment:
Có cách dùng toán tử 3 ngôi (ternary operator) để viết gọn cấu trúc `if...else` này, cụ thể là:
`page_title.empty? ? base_title : page_title + " | " + base_title`

Em có thể giữ nguyên cách viết như hiện tại hoặc dùng cách này nhé